Vagrantfile.example

```diff
@@ -60,6 +60,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
     dev.vm.provider :libvirt do |domain, override|
         domain.cpus = 4
+        domain.cpu_mode = "host-passthrough"
```

Comment:
Can a comment be added for this that links to an issue with a symptom description?